code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -684,7 +684,7 @@ Then we space some of our heat, and think about if we should stop conducting.
 		return
 	if(conductivity && thermal_conductivities[priority] != conductivity)
 		return
-	LAZYSET(thermal_conductivities, priority, null)
+	LAZYNULL(thermal_conductivities) //realistically we should only have one source
```

Comment:
if we only ever have one source there's no point in the system here, also isn't this what the lazynull at the end of the loop is supposed to do?

Reply:
The lazynull at the end of the loop set the list to null yes, but not before the turf thermal conduction is set to null. Should I change the logic and just apply and remove the thermal conductivity by storing it in a simple var instead of a list?

Reply:
thermal_conductivity isn't ever set to null, what would cause that?

The point of having this system in the first place is to support more then one thing modifying conduction, so people in future don't break it.

Though... Having said that, I may be an idiot, One moment